src/chat.cpp

```diff
@@ -594,7 +607,30 @@ bool CHAT_Input( event_t *pEvent )
 			}
 			else if ( pEvent->data1 == '\t' )
 			{
-				g_ChatBuffer.TabComplete();
+				if (( g_ulChatMode == CHATMODE_PRIVATE_SEND ) && ( g_ulChatPlayer != MAXPLAYERS ))
```

Comment:
I think it would be safer to also check `g_ulChatPlayer != consoleplayer` and `SERVER_CalcNumPlayers() >= 2`. In case both are false, the while loop won't terminate.

Reply:
I agree, I will add this check as well.